nailgun/entities.py

```diff
@@ -1018,7 +1018,7 @@ def create(self, create_missing=None):
         <https://bugzilla.redhat.com/show_bug.cgi?id=1223540>`_.
 
         """
-        return AbstractDockerContainer(
+        return self.__class__(
```

Comment:
Can you use `type(self)` instead of `self.__class__`? I'd prefer to avoid references to magic attributes.